pkg/generator/template.go

```diff
@@ -0,0 +1,76 @@
+package generator
+
+import (
+	"bytes"
+	"reflect"
+	"text/template"
+
+	"github.com/gophersbd/ormpb/pkg/descriptor"
+	"github.com/gophersbd/ormpb/protobuf"
+)
+
+type param struct {
+	*descriptor.File
+	Imports    []descriptor.GoPackage
+	ColumnTags []string
+}
+
+func applyTemplate(p param) (string, error) {
+	w := bytes.NewBuffer(nil)
+
+	p.ColumnTags = make([]string, 0)
+	t := protobuf.ColumnOptions{}
+	co := reflect.ValueOf(&t).Elem()
+	typeOfCO := co.Type()
+
+	for i := 0; i < co.NumField(); i++ {
+		p.ColumnTags = append(p.ColumnTags, typeOfCO.Field(i).Name)
+	}
+
+	if err := ormTemplate.Execute(w, p); err != nil {
+		return "", err
+	}
+	return w.String(), nil
+}
+
+var (
+	ormTemplate = template.Must(template.New("orm").Parse(`
+// Code generated by protoc-gen-orm. DO NOT EDIT.
+// source: {{ .GetName }}
+
+package {{ .GoPkg.Name }}
+
+const(
```

Comment:
Those constants values are jeneric? isn't it? May be move those to a `pkg/runtime/tags` package and import things from there??

Reply:
Yes. Right. OtherWise, This may conflict